client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -123,16 +123,24 @@ $deisgn-selector-selection-space: 175px;
 .page-layout-selector__grid {
 	display: grid;
 	grid-gap: 1.25em;
-	@include breakpoint( '>660px' ) {
-		grid-template-columns: 1fr 1fr 1fr;
-	}
+	grid-template-columns: repeat( auto-fill, minmax( 280px, 1fr ) );
 }
 
 .page-layout-selector__item {
 	position: relative;
 	overflow: hidden;
 	padding: 2px;
 
+	&::before {
+		content: '';
+		display: inline-block;
+		width: 1px;
+		height: 0;
+		padding-bottom: calc(
+			100% * 7 / 8
+		); // This gives us a 7:8 aspect ratio for individual grid items.
```

Comment:
We don't need `calc` here, that's a CSS `calc`, but we can let SASS compute it at build time.

This is sufficient:

```scss
&::before {
  padding-bottom: 7 / 8 * 100%; // This gives us a 7:8 aspect ratio for individual grid items.
}
```

That said, we seem to have an optimization step that detects static values in calc and replaces them with the calculated result, because I was surprised to check the build and find `87.5%` 🤷‍♂

Reply:
Oh, good to know!

Tempted to leave as-is, as I kinda prefer sticking closer to CSS syntax, and let optimization take care of it. (A very slim preference tho.)